docs/cudf/source/user_guide/index.md

```diff
@@ -0,0 +1,17 @@
+# User Guide
+
+```{toctree}
+:maxdepth: 2
+
+10min.md
+pandas-comparison.rst
+data-types.rst
+io.rst
+missing-data.md
+groupby.rst
+guide-to-udfs.md
+cupy-interop.md
+dask-cudf.rst
+internals.rst
+PandasCompat.rst
```

Comment:
Do these need to be updated? Some files appear to be missing, others were renamed.
```suggestion
data-types.md
io.md
missing-data.ipynb
groupby.md
guide-to-udfs.ipynb
cupy-interop.ipynb
dask-cudf.md
internals.md
PandasCompat.md
```